fix(changelog): update license, commit links in changelog and metrics

jzolnowski opened the pull request:
- fix links to commits in changelog
- remove unnecessary metrics
- fix `No files matching the pattern "'**/*.{js,ts}'" were found` when
  run lint (for Windows)

Commit:
docs(changelog): fix links to commits in changelog

Splaktar commented on CHANGELOG.md:
```diff
@@ -27,7 +27,7 @@ All notable changes to this project will be documented in this file. See [standa
 ### Features
 
-- **metrics:** collect http request metrics and package download metrics ([2cdebb9](https://github.com/xlts-dev/verdaccio-prometheus-middleware/commit/2cdebb93052605a47e2222a50a737df77a1548dd)), closes [#9](https://github.com/xlts-dev/verdaccio-prometheus-middleware/issues/9)
+- **metrics:** collect http request metrics and package download metrics ([7f26b37](https://github.com/DevIntent/verdaccio-prometheus-middleware/commit/7f26b37ee90f056ee07bfa4f760a2a084cb6d9c5)), closes [#9](https://github.com/DevIntent/verdaccio-prometheus-middleware/issues/9)
```
https://github.com/DevIntent/verdaccio-prometheus-middleware/issues/9 isn't going to work since the issues don't copy over to the new repo. This should continue to point to the old repo for the issue.

Splaktar replied:
Same issue with the other changes.

jzolnowski replied:
@Splaktar I don't have an access to Issues on this repo 😄

jzolnowski replied:
Fixed

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/DevIntent/verdaccio-prometheus-middleware?pullRequest=1) <br/>All committers have signed the CLA.

jzolnowski commented:
Not sure why unit tests failed. I did fix that issue [here](https://github.com/DevIntent/verdaccio-prometheus-middleware/pull/1/files#diff-f1316670f3c56889213957cd7d937cbe1937f3e28c271fc34435f1c6d41146da). Locally everything works properly 😄

Splaktar commented:
@jzolnowski please sign the CLA. I'm not sure what's going on with the tests.

Splaktar reviewed (changes requested):
Please break up the changes into 2 commits and update the changelog to the following
```
docs(changelog): fix links to commits in changelog
```
```
test(pluginMetricsDefault): remove unnecessary metrics

- fix `No files matching the pattern "'**/*.{js,ts}'" were found` when run lint
```
Is this only seen on Windows? If so, we should state that the fix is for Windows.

jzolnowski commented:
> Is this only seen on Windows? If so, we should state that the fix is for Windows.

Correct, but it's also consistent with other projects e.g. [verdaccio-google-cloud-storage](https://github.com/DevIntent/verdaccio-google-cloud-storage/blob/master/package.json#L68),  or [devintent](https://github.com/DevIntent/devintent/blob/master/package.json#L29)

Commit:
test(pluginMetricsDefault): remove unnecessary metrics

- fix `No files matching the pattern "'**/*.{js,ts}'" were found` when
  run lint (for Windows)

Splaktar commented on LICENSE.md:
```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2022 XLTS.dev
+Copyright (c) 2023 DevIntent
```
This doesn't render properly in markdown. This minor change will fix it.
```suggestion
- Copyright (c) 2022 XLTS.dev
- Copyright (c) 2023 DevIntent
```

Splaktar committed:
chore(license): update formatting

Signed-off-by: Michael Prentice <splaktar@devintent.com>

Splaktar committed:
chore(npm): fix critical audit warning

Splaktar committed:
chore(ci): update circleci/node orb

Splaktar committed:
chore(ci): update coverage path

Splaktar committed:
test(pluginMetricsDefault): add back removed metrics

- they exist in CI and fail without them

Splaktar commented:
From https://github.com/siimon/prom-client/blob/master/README.md#default-metrics:

> NOTE: Some of the metrics, concerning File Descriptors and Memory, are only available on Linux.

So CircleCI on Linux will have a different set of metrics compared to a local machine (macOS). @jzolnowski can you please update this test to run differently if on CircleCI?

Splaktar reviewed (changes requested):
Need to check different metrics if running in CircleCI.

jzolnowski commented:
> So CircleCI on Linux will have a different set of metrics compared to a local machine (macOS). @jzolnowski can you please update this test to run differently if on CircleCI?

I can look into it, but at this point I don't know the configuration of tests on this repository

Commit:
test(metrics-default): compare metrics depending on the operating system recognized

jzolnowski commented:
@Splaktar unit tests fixed 🎉

Splaktar commented on tests/pluginMetricsDefault.spec.ts:
```diff
@@ -65,7 +65,36 @@ describe('Default Metrics', () => {
       const metrics = await register.getMetricsAsJSON();
-      expect(metrics.map(({ name }) => name).sort()).toEqual([
+      // metrics order matters
+      const baseMetrix = [
```
```suggestion
      const baseMetrics = [
```

jzolnowski replied:
Fixed

Splaktar commented on tests/pluginMetricsDefault.spec.ts:
```diff
@@ -65,7 +65,36 @@ describe('Default Metrics', () => {
+        'process_start_time_seconds',
+      ];
+      // metrics order matters
+      const linuxMetrix = [
```
```suggestion
      const linuxMetrics = [
```

jzolnowski replied:
Fixed

Splaktar commented on tests/pluginMetricsDefault.spec.ts:
```diff
@@ -95,7 +124,8 @@ describe('Default Metrics', () => {
         'process_virtual_memory_bytes',
-      ]);
+      ];
+      expect(metrics.map(({ name }) => name).sort()).toEqual(process.platform === 'linux' ? linuxMetrix : baseMetrix);
```
```suggestion
      expect(metrics.map(({ name }) => name).sort()).toEqual(process.platform === 'linux' ? linuxMetrics : baseMetrics);
```

jzolnowski replied:
Fixed
